packages/matter.js/src/protocol/interaction/InteractionServer.ts

```diff
@@ -193,9 +193,8 @@ export class InteractionServer implements ProtocolHandler<MatterDevice> {
         );
 
         if (interactionModelRevision > INTERACTION_MODEL_REVISION) {
-            throw new StatusResponseError(
-                `Interaction model revision ${interactionModelRevision} not supported`,
-                StatusCode.InvalidAction,
+            logger.debug(
+                `Interaction model revision of sender ${interactionModelRevision} is higher than supported ${INTERACTION_MODEL_REVISION}.`,
```

Comment:
Lots of replication here, maybe a helper function?

Reply:
I also thought, but decided for now to leave that way until we have first specs infos and aybe the first "revision specific logics" and see how we need to hande it really